### PR TITLE
tables-moving to ORM?

### PR DESCRIPTION
**Pull request by aryehlev**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (simzacks) [inline .vscode/settings.json:1]: you'll want to add this file to .gitignore and not add it to the repo

File: bootstrap.sh [hunk 2]
```diff
@@ -9,3 +9,13 @@ export FLASK_APP=/vagrant/superx/app.py
 export FLASK_ENV=development
 
 flask run -h 0.0.0.0 -p 5000
+
+
+
+
+
```
Comment (simzacks) [inline bootstrap.sh:16]: export FLASK_ENV=development
flask run -h 0.0.0.0 -p 5000	flask run -h 0.0.0.0 -p 5000

are repeated in the file

File: bootstrap.sh [hunk 3]
```diff
@@ -9,3 +9,13 @@ export FLASK_APP=/vagrant/superx/app.py
...
+export FLASK_APP=/vagrant/app.py
+
+export FLASK_ENV=development
+
+flask run -h 0.0.0.0 -p 5000
```
Comment (simzacks) [inline bootstrap.sh:21]: end file with a new line

File: db_config.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+    # General Config
+    SECRET_KEY = environ.get('SECRET_KEY')
+    FLASK_APP = environ.get('FLASK_APP')
+    FLASK_ENV = environ.get('FLASK_ENV')
+
+    # Database
+    SQLALCHEMY_DATABASE_URI = environ.get("SQLALCHEMY_DATABASE_URI")
+    SQLALCHEMY_ECHO = False
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
```
Comment (simzacks) [inline db_config.py:20]: end file with a new line

File: superx/__init__.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
```
Comment (simzacks) [inline superx/__init__.py]: i don't see where flask_sqlalchemy was installed.

File: superx/__init__.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    app = Flask(__name__, instance_relative_config=False)
+    app.config.from_object('config.Config')
+
+    db.init_app(app)
+
+    with app.app_context():
+        from . import routes  # Import routes
+        db.create_all()  # Create sql tables for our data models
+
+        return app
```
Comment (simzacks) [inline superx/__init__.py]: end file with a new line

File: superx/__init__.py [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+def create_app():
+    """Construct the core application."""
+    app = Flask(__name__, instance_relative_config=False)
+    app.config.from_object('config.Config')
+
+    db.init_app(app)
+
+    with app.app_context():
+        from . import routes  # Import routes
```
Comment (simzacks) [inline superx/__init__.py]: this won't work, because you haven't defined the routes file yet

File: superx/models.py [hunk 8]
```diff
@@ -0,0 +1,43 @@
+"""Data models."""
+from . import db
+
+
+class User(db.Model):
```
Comment (simzacks) [inline superx/models.py]: class User should be for users, but you are using it for item_info. Maybe the class should be called ItemInfo

File: superx/models.py [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+        db.Integer,
+        nullable = False;
+    )
+    item_price_mega = db.Column(
+        db.Integer,
+        nullable = False;
+    )
+    
+    def __repr__(self):
+        return '<Item {}>'.format(self.item_name)
```
Comment (simzacks) [inline superx/models.py]: end file  with new line

File: superx/models.py [hunk 10]
```diff
@@ -0,0 +1,43 @@
+"""Data models."""
+from . import db
+
+
+class User(db.Model):
+    """Data model for user accounts."""
+
+    __tablename__ = 'item_info'
```
Comment (simzacks) [inline superx/models.py]: lets plan a design session for the database. This needs some work

File: templates/products.html [hunk 11]
```diff
@@ -0,0 +1,88 @@
...
+						</div>
+					</div>
+				</form>
+			</div>
+
+		{% endfor %}
+	
+	</div>
+</body>
+</html>
```
Comment (simzacks) [inline templates/products.html]: end file with a new line

File: templates/products.html [hunk 12]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
```
Comment (simzacks) [inline templates/products.html]: I don't know if this file will give you exactly what you want, but its probably a good place to start.

File: superx/models.py [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+    item_unit_of_measure= db.Column(
+        db.String,
+        nullable = False;
+    )
+    item_unit_of_measure_qty= db.Column(
+        db.Integer,
+        nullable = False;
+    )
+
+    __tablename__ = 'item_prices'
```
Comment (simzacks) [inline superx/models.py]: each table needs its own class

**Review by simzacks — CHANGES_REQUESTED** [review]

File: superx/models.py (hunk 10 above)
Comment (aryehlev) [reply superx/models.py]: i understand that you did a session with yoav, i will sit with him later and try to make progress

**Pull request closed without merging**